tools/cbpManager.deploy/.env

```diff
@@ -0,0 +1,8 @@
+#Network configuration
+PORT=8081
```

Comment:
This port is already in use for the `-dev` Version of cBioPortal. So maybe another port may be more reasonable. I would suggest to use `8180` to avoid conflicts.